docs/docs/02-concepts/01-preflight-and-inflight.md

```diff
@@ -334,6 +334,83 @@ Within the first clause of the `lift` block, a list of qualifications on preflig
 Statements within a `lift` block are exempt from the compiler's analyzer that tries to determine preflight object usage automatically.
 If an inflight method is directly or indirectly called within a `lift` block without sufficient resource qualifications, it may result in errors at runtime.
 
+## Building liftable abstractions
+
+Compute environments where inflight code is executed are called **inflight hosts**.
```

Comment:
Should we have a link to inflight hosts somewhere? Or I think as you said move the title to reflect inflight hosts. First time the term is used, maybe the section should lead with this in the title.

Reply:
I updated the title -- I think this is the first time we introduce the concept or make it explicit. It could be worth giving the article a broader refresh in the future and see if the concept should be introduced earlier, not sure